.github/workflows/ee_latest_snapshot_push.yml

```diff
@@ -39,6 +40,7 @@ jobs:
       - name: Build Test EE image
         run: |
           docker buildx build --load \
+            --build-arg JDK_VERSION=${{ matrix.jdk }} \
             --build-arg HZ_VERSION=${{ github.event.inputs.HZ_VERSION }} \
             --build-arg HZ_VARIANT=${{ matrix.variant }} \
             --tag hazelcast-ee:test \
```

Comment:
Use jdk variable in `Store docker logs as artifact` and `Get docker logs`

Reply:
done